src/descriptors/power_system_structs.json

```diff
@@ -221,6 +221,7 @@
         {
           "null_value": "init",
           "name": "name",
+          "exclude_setter": true,
```

Comment:
People usually copy/paste these objects when creating new structs. You all will need to remember to ensure people add this field.